### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aandroide**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,7 +218,8 @@ Insert your Project Id and Model Id in quotes in the mentioned places
 
 1. Click [here](https://developer.spotify.com/dashboard/login) and register for a spotify developer account, if you already don't have one.  
 2. In the developer's dashboard, choose "**CREATE A CLIENT ID**". In the pop-up window provide the requested details.  
-3. Click on the new app created and copy the CLIENT ID and CLIENT SECRET. Paste it in the config.yaml file in the indicated space.  
+3. Click on the new app created and copy the CLIENT ID and CLIENT SECRET. Paste it in the config.yaml file in the indicated space.
+4. Access spotify:[here]( https://www.spotify.com/it/account/overview/) and copy the username to be entered in config.yaml
```
Comment (shivasiddharth) [inline README.md]: this redirects to Italian site.

Comment (aandroide) [reply README.md]: it is already true.
have you tried only with www.spotify.com?

Comment (shivasiddharth) [reply README.md]: it should be www.spotify.com/account/overview/ it will pick the location automatically.

Comment (shivasiddharth) [reply README.md]: Can you change that ? I shall merge the pull request.

Comment (aandroide) [reply README.md]: hello sid, I proceeded to correct the link

**Pull request merged**